.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,74 @@
+name: Deploy to dev
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [$default-branch]
+
+jobs:
+  deploy-dev:
+    name: Deploy master to dev server
+    runs-on: ubuntu-latest
+    environment: dev
+    concurrency: dev
+    env:
+      SETTINGS_PATH: ${{ github.workspace }}/OTP/mbta/semaphore/custom-settings.xml
+    
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ${{ github.workspace }}/OTP
+
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 8
+          settings-path: ${{ github.workspace }}/OTP/mbta/semaphore
```

Comment:
Just curious, did you try using ASDF instead? I don't actually know enough about Java to say if that would have any concrete advantages or disadvantages over this approach.

Reply:
Also regarding `settings-path`, should we rename that directory to something that doesn't mention Semaphore?

Reply:
I suppose the advantage of `actions/setup-java` is that it's presumably written by people who know more about Java than we do, but on the other hand if we take this approach that's one more little spot in the repo we'd have to update if we changed anything about the Java version going forward.

Reply:
The `asdf-vm` family of actions aren't set up to handle subdirectories (and this action is a bit unconventional, loading OTP into one folder and the onebusaway modules into a sibling directory), so I'm not sure I'd be able to get ASDF to work easily for this workflow anyway. Would you be OK leaving these actions to use `actions/setup-java` for now, and update them to use ASDF at a future point?

Reply:
Yeah that's fine by me!